sys/log/full/include/log/log.h

```diff
@@ -260,11 +268,11 @@ struct log *log_find(const char *name);
  * @brief Writes the raw contents of a flat buffer to the specified log.
  *
  * NOTE: The flat buffer must have an initial padding of length
- * `LOG_ENTRY_HDR_SIZE`.  This padding is *not* reflected in the specified
+ * `LOG_BASE_ENTRY_HDR_SIZE`.  This padding is *not* reflected in the specified
```

Comment:
I believe this comment is no longer accurate.  The buffer must contain enough padding to accommodate a full log entry header (which is more than `LOG_BASE_ENTRY_HDR_SIZE` if this new feature is enabled). Would you   mind changing this comment to reflect that?  I think it would be fine to just replace `LOG_ENTRY_HDR_SIZE` with `header-size` or something like that.

I only ask because these "non-body" functions have a confusing enough API as it is!

This comment applies to the doxygen comments for all the non-body functions in this file.